lib/lunchbot/lunchbot_data/order.ex

```diff
@@ -22,7 +22,13 @@ defmodule Lunchbot.LunchbotData.Order do
 
   def get_total(%Lunchbot.LunchbotData.Order{} = order) do
     # total price of an Order the the price of all the items in the order.
-    Enum.map(order.order_items, &Lunchbot.LunchbotData.OrderItem.get_total/1)
-    |> Enum.sum()
+    list_order_totals = Enum.map(order.order_items, &Lunchbot.LunchbotData.OrderItem.get_total/1)
+
+    # check for nil values, return nil if found
+    if is_nil(Enum.find(list_order_totals, true, fn x -> is_nil(x) end)) do
```

Comment:
could use `Enum.member?` here also.  E.G. `Enum.member?(["a", "b", nil], nil)` returns true.

Reply:
Yup, thanks; that does the functionality I was arranging awkwardly with that `Enum.find` call lol. Still learning all these different Enum methods.